DomainResponse.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Wrapper class for Custom Domain information
+ */
+class DomainResponse {
```

Comment:
Nitpick: `DomainInfo` might have been a better name after looking back at how we are using it

Reply:
Fixed.